docker/aarch64-linux-gnu-glibc.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -x
+set -euo pipefail
+
+main(){
+    yum install -y epel-release
+    yum install -y gcc-aarch64-linux-gnu gcc-c++-aarch64-linux-gnu binutils-aarch64-linux-gnu binutils gcc-c++ glibc-devel
+    yum clean all
+     
+    local td
+    td="$(mktemp -d)"
+  
+    pushd "${td}"
+    
+    local target=aarch64-linux-gnu
+    local prefix="/usr/${target}"
+    local kernel_v4="4.18.20"
+    
+    curl --retry 3 "https://mirrors.edge.kernel.org/pub/linux/kernel/v4.x/linux-${kernel_v4}.tar.xz" -O
+    tar -xvf "linux-${kernel_v4}.tar.xz"
+    cd "linux-${kernel_v4}"
+    make ARCH=arm64 INSTALL_HDR_PATH="${prefix}" headers_install
+    cd ..
+    
+    curl --retry 3 http://ftp.gnu.org/gnu/glibc/glibc-2.17.tar.xz -O
+    tar -xvf glibc-2.17.tar.xz
+    mkdir build && cd build
+    CC=/usr/bin/aarch64-linux-gnu-gcc \
+        LD=/usr/bin/aarch64-linux-gnu-ld \
+        AR=/usr/bin/aarch64-linux-gnu-ar \
+        RANLIB=/usr/bin/aarch64-linux-gnu-ranlib \
+        ../glibc-2.17/configure \
+        --prefix="${prefix}" \
+        --build="${MACHTYPE}" \
+        --host="${target}" \
+        --target="${target}" \
+        --with-arch=aarch64 \
+        --with-headers="${prefix}/include" \
+        --libdir="${prefix}/lib" \
+        --libexecdir="${prefix}/lib"
+    
+    make -j && make install
+    cd ..
+    
+    mkdir libgcc && cd libgcc
+    curl --retry 3 http://mirror.centos.org/altarch/7/os/aarch64/Packages/libgcc-4.8.5-44.el7.aarch64.rpm -O
+    rpm2cpio libgcc-4.8.5-44.el7.aarch64.rpm | cpio -idmv
+    mv lib64/* "${prefix}/lib"
+    ln -s "${prefix}/lib/libgcc_s.so.1" /usr/lib/gcc/aarch64-linux-gnu/4.8.5/libgcc_s.so
+    cd ..
+    
+    popd
+
+    rm -rf "${td}"
+    
```

Comment:
This is very small, but this should include `rm "${0}"`, so it removes the script after it finishes running, to avoid polluting to root directory.